rtlib/include/rt/ImageStitcher.hpp

```diff
@@ -0,0 +1,93 @@
+#pragma once
+
+#include <vector>
+#include <iostream>
+
+#include <opencv2/stitching.hpp>
+
+namespace rt
+{
+
+class ImageStitcher
+{
+public:
+    ImageStitcher() = default;
+
+    // Sets the images to be stitched
+    void setImages(cv::InputArrayOfArrays i) { i.getUMatVector(imgs_); }
```

Comment:
```suggestion
    void setImages(std::vector<cv::Mat> i);
```
- Interface should use STL types where possible. Consider storing as `cv::Mat` right here, and converting to vector of `cv::UMat` inside `compute()`. 
- Move the implementation into the cpp file.